### PR TITLE
change -L DEBUG to -V , issue #2688

### PR DESCRIPTION
**Pull request by mgracer48**
I changed the -L DEBUG to -V 
Good?

### CODE REVIEW

**Comment by sils1297** [comment]
As per :point_up: [September 15, 2016 6:59 PM](https://gitter.im/coala-analyzer/coala?at=57dad365df21a75730fddd50) this doesn't solve the issue as intended. (Though it's our fault.)

**Comment by sils1297** [comment]
We still want fbc6ac6 - but I would suggest changing the commit message.  Don't worry, most people don't know about good commit messages when they join us, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - maintainers will use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading <http://coala.io/commit>!

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

**Comment by sils1297** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*) This will get rid of this merge commit.

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

File: coalib/bears/Bear.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def execute(self, *args, **kwargs):
             return [] if result is None else list(result)
         except:
             self.warn(
-                "Bear {} failed to run. Take a look at debug messages (`-L "
+                "Bear {} failed to run. Take a look at debug messages (`-V "
                 "DEBUG`) for further information.".format(name))
```
Comment (sils1297) [inline coalib/bears/Bear.py]: `-V DEBUG`? `-V` doesn't need an argument so we'll want to remove the `DEBUG` from the command proposal.

This is weird to phrase, was this understandable? :/

Comment (mgracer48) [inline coalib/bears/Bear.py]: Nah i just made a dumb mistake whoops XD

**Comment by mgracer48** [comment]
Dang it -_-

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2760/commits/c236912af5f6d73ed9e543a97067e547aa28cc5f

File: coalib/bears/Bear.py [hunk 2]
```diff
@@ -250,8 +250,8 @@ def execute(self, *args, **kwargs):
             return [] if result is None else list(result)
         except:
             self.warn(
-                "Bear {} failed to run. Take a look at debug messages (`-L "
-                "DEBUG`) for further information.".format(name))
+                "Bear {} failed to run. Take a look at debug messages (`-V `)
```
Comment (sils1297) [inline coalib/bears/Bear.py]: missing closing `"`

**Comment by sils1297** [comment]
You can change your last commit without a rebase by just using `git commit --amend -a` you merge your current changes into it and can change the message as well.

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2760/commits/0c7754460d3008fb449faa66869416eb861611c3

**Comment by sils1297** [comment]
hey, fixing the test should be in the same commit as changing the functionality so we can later see the changes together. You can squash them together by doing the rebase interactive thing again and tell git to use "fixup" on the last commit to merge the two of them together keeping the first commit message. Shouldn't be too complicated.

**Comment by mgracer48** [comment]
Sigh no offense but why does it matter

**Comment by sils1297** [comment]
> Sigh no offense but why does it matter

No offense taken.

- What is in which commit matters mostly for debugging and clean history.
    - Your first commit changes functionality and breaks our builds, a revert e.g. because an issue arises would break our build.
    - `git blame` is the best documentation your code has. If you want to know more about a LOC use git blame. Clean commits make it easier to use.
    - `git bisect` relies on history as well, breaking code in one commit and fixing it in another can interrupt a `git bisect run` command.
    - It's easier to review - less bugs slip through.
- The newcomer issues are here to show you the ideal process. With newcomers we're not sure how long you'll be staying around to help us debug issues that come up with your code - we'll be much more strict with newcomers because of this; you'll see this behaviour in a lot of OS projects - long term contributors can make a mess (at least a bit) but they're also expected to clean it up. I personally consider teaching/learning the most important aspects of our newcomer program.

**Comment by sils1297** [comment]
see also http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#why-do-we-need-good-commits , be sure to file an issue if it doesn't make sense to you. We usually offer short term contributors to clean up commits for them if they don't plan to stick longer than a few commits with us.

**Comment by sils1297** [comment]
@underyx we're getting the timeout thing again, it's not his fault, any idea? Just increase the timeout?

**Comment by sils1297** [comment]
Ping us if there are doubts left of course.

**Comment by sils1297** [comment]
(Or if you want us to clean it up for you)

**Comment by mgracer48** [comment]
im busy so i won't be able to work on this for a while

**Comment by mgracer48** [comment]
ill try to do it tomorrow or later tonight

**Comment by mgracer48** [comment]
done

**Comment by sils1297** [comment]
Hi, we dislike the merge commits because our history would be cluttered by more merge commits than anything else; instead we use a rebase. Both your commits are good, if you're around you can do another quick rebase, otherwise I can do it for you.

**Comment by sils1297** [comment]
ack https://github.com/coala/coala/pull/2760/commits/ccc37a526aa8819c3748f8dcbcda07486dd1d084

**Comment by sils1297** [comment]
ack https://github.com/coala/coala/pull/2760/commits/857df0e855c7a9cb0dffdfdd515805c9629bf8e2 https://github.com/coala/coala/pull/2760/commits/7a1f395b162b64e6c4366298dedc17a8563e2dd2

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**